google/cloud/spanner_dbapi/cursor.py

```diff
@@ -207,9 +213,31 @@ def fetchone(self):
         self._raise_if_closed()
 
         try:
-            return next(self)
+            res = next(self)
+            self._checksum.consume_result(res)
+            return res
         except StopIteration:
-            return None
+            return
+        except Aborted:
+            self.connection.retry_transaction()
+            return self.fetchone()
```

Comment:
When we retried a transaction up to the failed row, we need to re-execute the failed `fetchone()`